src/seedu/addressbook/data/person/Email.java

```diff
@@ -54,8 +54,17 @@ public int hashCode() {
         return value.hashCode();
     }
 
-
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    /**
+     * Implements the getPrintableString() method from the Printable interface
```

Comment:
- no need to mention the method name again here. 
- you can choose to provide a proper header comment to the interface and skip comment in each implementation